backend/lib/services/projects.js

```diff
@@ -148,23 +151,14 @@ exports.list = async function ({ user, qs = {} }) {
     return hasGroupMembership || hasUserMembership || hasServiceAccountMembership
   }
 
-  const phases = _
-    .chain(qs)
-    .get('phase', 'Ready')
-    .split(',')
-    .compact()
-    .value()
   return _
     .chain(projects)
     .filter(project => {
       if (!isAdmin && !isMemberOf(project)) {
         return false
       }
-      if (!_.isEmpty(phases)) {
-        const phase = _.get(project, 'status.phase', 'Initial')
-        return _.includes(phases, phase)
-      }
-      return true
+      const phase = _.get(project, 'status.phase', 'Pending')
```

Comment:
This needs to be aligned with the #1246.

Reply:
If we introduce a reusable function `filterProjects` maybe in `utils/index.js` we could reuse it here and in my PR.
```js
function filterProjects (projects, user, isAdmin = false) {
  const isMemberOf = project => {
    const hasGroupMembership = _
      .chain(project)
      .get('spec.members')
      .filter(['kind', 'Group'])
      .map('name')
      .intersection(user.groups)
      .size()
      .gt(0)
      .value()

    const hasUserMembership = _
      .chain(project)
      .get('spec.members')
      .filter(['kind', 'User'])
      .map('name')
      .includes(user.id)
      .value()

    const hasServiceAccountMembership = _
      .chain(project)
      .get('spec.members')
      .filter(['kind', 'ServiceAccount'])
      .map(({ namespace, name }) => `system:serviceaccount:${namespace}:${name}`)
      .includes(user.id)
      .value()

    return hasGroupMembership || hasUserMembership || hasServiceAccountMembership
  }

  return _
    .chain(projects)
    .filter(project => {
      if (!isAdmin && !isMemberOf(project)) {
        return false
      }
      const phase = _.get(project, 'status.phase', 'Pending')
      return phase !== 'Pending'
    })
    .value()
}
```

This function would look like:
```js
exports.list = async function ({ user }) {
  const isAdmin = await authorization.isAdmin(user)
  const projects = filterProjects(cache.getProjects(), user, isAdmin)
  return _.map(projects, fromResource)
}
```

Reply:
@holgerkoser will do in his PR

Reply:
done https://github.com/gardener/dashboard/pull/1246/commits/7eecd1653eebce9a29269dec5da54b9e994dc288